ci_tools/bumpversion.sh

```diff
@@ -0,0 +1,5 @@
+bump2version patch --message "Bump version: {current_version} to {new_version} [ci skip]"
```

Comment:
What does  "[ci skip]" do?

Reply:
This skips travis on the commit which updates the version on master.
Without this tag, Travis gets stuck in an infinite loop of:
(1) Update version
(2) This triggers a build
(3) Which updates version
(4) Which triggers another build
(5) And so on...

https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build